Xamarin.PropertyEditing.Tests/MockControls/MockNSButton.cs

```diff
@@ -21,6 +23,7 @@ public MockNSButton ()
 			AddProperty<NSCellStateValue> ("State");
 			AddProperty<string> ("Title");
 			AddProperty<bool> ("Transparent");
+			AddProperty<CommonBrush> ("BackgroundColor");
```

Comment:
Small detail, but the properties have been added in alphabetical order so far.

Reply:
done